draft-ietf-httpbis-http2-encryption.md

```diff
@@ -64,16 +64,18 @@ for this draft can be found at <https://github.com/httpwg/http-extensions/labels
 # Introduction
 
 This document describes a use of HTTP Alternative Services {{RFC7838}} to decouple
-the URI scheme from the use and configuration of underlying encryption, allowing a `http` URI
-{{RFC7230}} to be accessed using Transport Layer Security (TLS) {{RFC5246}} opportunistically.
-
-Serving `https` URIs requires avoiding Mixed Content {{W3C.CR-mixed-content-20160802}}, which is
-problematic in many deployments. This document describes a usage model whereby sites can serve
-`http` URIs over TLS, thereby avoiding these issues, while still providing protection against
+the URI scheme from the use and configuration of underlying encryption.
+It allows HTTP/2 {{RFC7540}} to access a `http` URI
```

Comment:
This is worded oddly. Suggest "It allows a `http` URI to be accessed using HTTP/2 {{RFC7230}} and Transport Layer..."